src/components/ui/card/card-info.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from "react"
+
+import {
+  Card,
+  CardDescription,
+  CardHeader,
+  CardTitle,
+} from "@/components/ui/card/card"
+import { THeaderProps, TSubtitleProps, TTitleProps } from "./index.type"
+
+export interface ICardInfoProps {
+  title: string
+  subTitle?: string
+  titleProps?: TTitleProps
+  headerProps?: THeaderProps
+  subTitleProps?: TSubtitleProps
+}
+
+export function CardInfo(props: ICardInfoProps) {
+  const { title, subTitle, titleProps } = props
+  return (
+    <Card className="min-w-[100px]">
```

Comment:
Seem like this card is not round enough 
![image](https://github.com/blackmouse572/quizler-fe/assets/77144199/1c45b7a1-8bed-43d5-885a-58f7b8c0c94f)